src/main/java/com/cascadebot/cascadebot/commands/core/ServerInfoCommand.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2018 CascadeBot. All rights reserved.
+ * Licensed under the MIT license.
+ */
+
+package com.cascadebot.cascadebot.commands.core;
+
+import com.cascadebot.cascadebot.commandmeta.CommandContext;
+import com.cascadebot.cascadebot.commandmeta.CommandType;
+import com.cascadebot.cascadebot.commandmeta.ICommand;
+import com.cascadebot.cascadebot.messaging.MessagingObjects;
+import com.cascadebot.cascadebot.utils.FormatUtils;
+import net.dv8tion.jda.core.EmbedBuilder;
+import net.dv8tion.jda.core.entities.Member;
+import java.util.Set;
+
+public class ServerInfoCommand implements ICommand {
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+
+        EmbedBuilder builder = MessagingObjects.getInfoEmbedBuilder();
+        builder.setTitle(context.getGuild().getName());
+        builder.setThumbnail(context.getGuild().getIconUrl());
+        builder.setDescription("Guild Information");
+        builder.addField("Creation Date", FormatUtils.formatDateTime(context.getGuild().getCreationTime()), true);
+        builder.addField("Guild Name", context.getGuild().getName(), true);
+        builder.addField("Owner", context.getGuild().getOwner().getUser().getAsTag(), true);
+        builder.addField("Region", context.getGuild().getRegion().toString(), true);
+        builder.addField("Member Count", String.valueOf(context.getGuild().getMembers().size()), true);
+        builder.setFooter("ID: " + context.getGuild().getId(), context.getGuild().getIconUrl());
+
+        context.replyInfo(builder);
```

Comment:
Just use reply

Reply:
Ummm no? we want this to use our new system do we not?